sale_ux/README.rst

```diff
@@ -25,6 +25,7 @@ Several Improvements to sales:
 #. Add button to force "invoiced" only for admin with tec features.
 #. Add tracking to payment term and pricelist on sale orders.
 #. Add option to show SO reference field on tree view and on main section of form view.
+#. Add filters contracts on sales orders so you can only select the ones with same partner as the sales order.
 
```

Comment:
Update doc, not mention contract, mention both partner/commercial partner

```diff
-#. Add filters contracts on sales orders so you can only select the ones with same partner as the sales order.
+#. Add filter to be able to select analytic account on sale orders with the same partner/commercial partner as the sale order.  
```
Add new item

#. Add Commercial Partner is automatic set in the sale order taking into account the partner configuration